main_test.go

```
@@ -677,6 +678,50 @@ func publishBenchMessage(b testing.TB, queueID string, body []byte) {
+
+func TestAckRatePerSec_MatureQueue(t *testing.T) {
+	m := &queueMetrics{
+		startedAt: time.Now().Add(-120 * time.Second),
+	}
+	for i := 0; i < 60; i++ {
+		m.ackWindow = append(m.ackWindow, time.Now())
+	}
+
+	rate := m.ackRatePerSec()
+
+	expected := 60.0 / 60.0
```

golangci-lint (2.11.4)

[error] SA4000: identical expressions on the left and right side of the '/' operator

(staticcheck)